### DISCUSSION TITLE
Maçın orta hakemi kimdir sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#221] [Zeka Sorusu] [answered]
A, B ve C bir futbol maçında hakemlik yapmaktadır.

A ve B'nin kısa olanı, yan hakemlerin yaşlı olanıdır.
B ve C'nin genç olanı, yan hakemlerin kısa olanıdır.
A ve C nin uzun olanı, yan hakemlerin genç olanıdır.
Maçın orta hakemi kimdir?

**Comment by Adaisland** [comment, marked as answer]
Yukarıdaki yönergeleri okuyunca ilk iki yönergeden 3 tür insan belirleyebiliriz:
1.En yaşlı
2.yaşlı
3.genç
- ilk iki yönergede de ortak olarak "B" kısa yönergesinde geçiyor yani"B"nin uzun olma şansı yok ve ilk ifadeden de"B"nin yaşlı yan hakem olduğunu görüyoruz o zaman "C" kişisinede en yaşlı düşüyor ve "A" yada bunuda son yönergedeki uzun ve genç ifadesinden çıkarıyoruz çünkü "A""B"den küçük "B" ise "C" den küçük

Bu durumda orta hakem en yaşlı yani "C" kişisi oluyor
